tools/jarcat/tar/tar_test.go

```diff
@@ -0,0 +1,105 @@
+package tar
+
+import (
+	"archive/tar"
+	"bytes"
+	"compress/gzip"
+	"io"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+const testDataDir = "tools/jarcat/tar/test_data"
```

Comment:
On ubuntu 16+ `/tmp` should be mounted as `tmpfs` by default; this is also the case for the Fedora-family and ArchLinux but I don't know about others. 
I forgot that plz would do this for you, so it's fine as is;